### DISCUSSION TITLE
c3OS rename/branding

### DISCUSSION THREAD

**Discussion by mudler** [kairos-io/kairos#84] [Polls]
Hi!

Initially when I've created c3os, I've thought at the name as containerOS^3 (kubed ;)), hence c3OS. 

Although, while it was fine just fine to begin with, c3OS has several drawbacks:

- it contains a number in the name - that makes it unpractical to type, spell, and go as well into the code base ( a CRD c3OSCluster, is both hard to type, clashes visually in the codebase, etc)
- It contains OS - while it sounds nice, and the trend seems to not care about the difference about a distribution and an OS - I'd like to bring back some normality and avoid OS in the name. Yes, c3os at the end of the day is an OS, but it boils down to a Linux (meta)Distribution.
- it is too much a k3os reminder. While c3OS shares most of the project goals with k3os, it is a completely different project

As this project is starting to grow, and it is a community driven project - we need to settle on a name for the project going forward, and I'd like this process to be as open and transparent as possible with constructive criticism.

I'm trying to collect feedback here on name proposals, and I'd go ahead by picking the name with majority of votes, although reserving rights to pick a final decision ( just to avoid this to steer badly ). Poll ends up 28th of August. If polls have no outcomes, I'll go ahead and pick one.
 
Keep in mind some aspects while picking up/proposing:
- it is an immutable distro
- focused on running kubernetes/container workloads
- containers are used to run the OS
- allows to bring your own distro, turning them in an A/B immutable style, cloud-ready
- optional fully-meshed p2p support, with automated node setup

Feel free to propose other names as well which are not part of the poll, and thank you so much for your time!

**Comment by antongisli** [comment]
"Compose" or "Edge compose" ?

**Comment by codrin-iftimie** [comment]
This should shed some light on [Kairos](https://twitter.com/culturaltutor/status/1558551902505279489)

**Comment by mudler** [comment]
Kairos is unfortunately already taken (also a company with the same name), we are currently iterating over Kairos similar names

e.g. keirros

**Comment by mudler** [comment]
Closing poll. Kairos (w/ variants) won
